bin/hipify-perl

```diff
@@ -6159,6 +6159,7 @@ sub simpleSubstitutions {
     subst("cuda.h", "hip\/hip_runtime.h", "include_cuda_main_header");
     subst("cuda_runtime.h", "hip\/hip_runtime.h", "include_cuda_main_header");
     subst("cudnn.h", "hipDNN.h", "include_cuda_main_header");
+    subst("cutensor.h", "hiptensor.h", "include");
```

Comment:
Is include the correct choice here or is it include_cuda_header?

Reply:
Nope, as `hipify-perl` is an entirely autogenerated script (`hipify-clang --perl`): https://github.com/ROCm/HIPIFY/blob/amd-staging/bin/hipify-perl#L25.
You should change its generation, not the script itself.